### ISSUE TITLE
Autogenerated cards put the card translation into the explanation field

### ISSUE THREAD

**Issue by deleuzei** [guiferviz/mnemocards#30]
As you can see in the screenshot, on Google Translate the box under the translated word features words in both languages.

![image](https://user-images.githubusercontent.com/92121313/148450370-897a1ce5-3372-45c1-ad04-82ad6802bd03.png)

Autogenerating fetchs the entire box, resulting in both translations appearing in the explanatory field.
![image](https://user-images.githubusercontent.com/92121313/148449708-34f2105a-da2c-4d28-8bae-70cda72bb585.png)

![image](https://user-images.githubusercontent.com/92121313/148449601-405cbd69-18d8-44d6-842d-4799f820e366.png)

YourLanguageExplanation field should be purged of all references to the foreign language, else it is liable that the word you are supposed to translate will appear there.

**Comment by Otter-man** [comment]
@deleuzei hi again!
It seems you created an autogenerated TSV and then generated cards with a `vocabulary` set to `type`, that's why you see the entire box without the 'spoilers' hidden. 
Instead, you should set `type` for autogenerated TSV to `autogenerate`, and set `file` to your autogenerated TSV-file.
The card will be generated with 'spoilers' hidden until you [flip the card, like in this example.](https://user-images.githubusercontent.com/59960096/148457322-ffc25d01-f6fd-4d38-bbb5-e774ee6affcb.png).

The config should look something like this (there is no option for header since autogenerated TSVs are assumed to have a header.

```json

    {
        "type": "autogenerate",
        "file": "en_fr_autogenerated.tsv",
        "card_color": "#33AA33",
        "show_tags": true,
        "card_properties": {
            "tags": [
                "french",
                "vocabulary",
                "autogenerated"
            ]
        },
        "pronunciation_in_reverse": false,
        "lang": {
            "original": "en",
            "translation": "fr"
        },
        "one_translation": false,
        "audio": true
    }
```

**Comment by deleuzei** [comment]
After further testing I determined that this occurs only when autogenerating cards with the intermediary step of creating a .tsv file. If the cards are marked autogenerate, then the synonyms are revealed only after the card is turned, if the cards are generated from a file created using maketsv, then the synonyms are visible immediately.

**Comment by deleuzei** [comment]
You're completely right, I've missed the part that you're still supposed to use the autogenerate type, even though you're importing from a .tsv. 
@Otter-man There's one problem with using that method though. If using autogenerate type, the JSON audio field doesn't recognize a media_dir subfield. Consequently the audio is downloaded to ./.media. Is there a way around this, beside building in one place and moving the artifacts elsewhere? Have I missed something else in the readme?

**Comment by Otter-man** [comment]
@deleuzei nope, configs for `autogenerate` don't support saving media to a non-default folder yet, so the only way currently is to move/delete the default `.media` folder manually.
